src/freertos/posix/include/freertos/freertos_platform_config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __FREERTOS_PLATFORM_CONFIG_H_
```

Comment:
Why does this file exists twice? See  src/freertos/posix/config/include/freertos_platform_config.h

Reply:
Removed the duplicate. Should only have been in the posix folder. It was not included by the CMakeLists